Sming/SmingCore/ElapseTimer.h

```diff
@@ -0,0 +1,41 @@
+/****
```

Comment:
The ElapseTimer class has to prove itself. Move it for the moment as part of the sample. Later one we can make it part of the core if ElapseTimer is used a lot in other parts of the framework.

Reply:
I uses this in other places as a profiling aid, not so much in release code. It would be helpful to me if it could be more accessible; could it go in Services, perhaps?

Reply:
> could it go in Services, perhaps?

Which reminds me that we should refactor Services and name it to something like "Extras" + Move libbase64 to third-party + move WebHelers to SmingCore, etc.

Ok, please move ElapsedTime to Services for now.

Reply:
I'll stick it in a folder called 'Profiling' ?

Reply:
Ok, make sure the Profiling is also checked for coding style issues.